src/components/AccountLoginForm.jsx

```diff
@@ -1,14 +1,26 @@
-import React from 'react'
+import styles from '../styles/accountLoginForm'
 
+import React from 'react'
+import classNames from 'classnames'
 import Field, { PasswordField, DropdownField, CheckboxField } from './Field'
 
-const AccountLoginForm = ({ t, fields }) => (
-  <div className='account-form-login'>
+const AccountLoginForm = ({ t, konnector, customView, fields, error, dirty, submitting, onSubmit }) => (
+  <div className={styles['account-form-login']}>
+    {customView &&
+      <div className='coz-custom-view'
+        dangerouslySetInnerHTML={{
+          __html: customView.replace(/<%t (.*) %>/gi, (match, $1) => t($1))
```

Comment:
It is part of #71, but it's a legacy of the previous code version. Some konnectors have a `customView` property which has to be displayed (It is the case of Orange Mobile I think).